content/en/Getting started/what-to-expect.md

```diff
@@ -55,7 +55,8 @@ results. Here's what you can expect:
 1. We keep the Slack channel open until you've set each finding to:
    - Accepted Risk
    - Fixed
-   If you need access to the archived Slack channel, contact your Customer Success Manager or support@cobalt.io.
+   <br>{{% contact-csm-support %}} if you need access to the archived Slack channel.
```

Comment:
@mjang-cobalt I changed the formatting because the next ordered item appeared as `1`.